plugins/statistics.koplugin/main.lua

```diff
@@ -598,6 +603,39 @@ function ReaderStatistics:upgradeDBto20201022(conn)
     conn:exec("PRAGMA user_version=20201022;")
 end
 
+function ReaderStatistics:upgradeDBto20221022(conn)
+    conn:exec([[
+        -- We make the index on book's (title, author, md5) unique in order to sync dbs
+        -- First we fill null authors with '' and remove duplicates
+        UPDATE book SET authors = '' WHERE authors IS NULL;
+        UPDATE page_stat_data SET id_book = (
```

Comment:
May be add a comment line for the more complex 2nd UPDATE (+ its DELETE): `-- Secondly, ...`
And add a blank line to separate actions - or remove the blank line above `-- Finally` just to make all this nice SQL consistent and easier to grasp.

Reply:
Done.